requirements.txt

```diff
@@ -5,7 +5,7 @@ django-cas-ng
 pucas>=0.8
 eulxml>=1.1.3
 viapy>=0.2
-wagtail>=2.16,<2.17
+wagtail>=5.1,<5.2
 # pin for js error with dal>3.5; see https://github.com/Princeton-CDH/mep-django/issues/716
```

Comment:
I think this comment can be removed, it isn't accurate anymore  - that issue is one that Josh will need to test before we can release, but the requirement has already been updated.  (Not your changes but wanted to fix while I see it).
```suggestion
```